otlp/common.go

```diff
@@ -260,16 +263,13 @@ func addAttributesToMap(attrs map[string]interface{}, attributes []*common.KeyVa
 		if attr.Key == "" || attr.Value == nil {
 			continue
 		}
-		if val, truncatedBytes := getValue(attr.Value); val != nil {
-			attrs[attr.Key] = val
-			if truncatedBytes != 0 {
-				// if we trim a field, add telemetry about it; because we trim at 64K and
-				// a whole span can't be more than 100K, this can't happen more than once
-				// for a single span. If we ever change those limits, this will need to
-				// become additive.
-				attrs["meta.truncated_bytes"] = val
-				attrs["meta.truncated_field"] = attr.Key
-			}
+		if truncatedBytes := flatten(attrs, attr.Key, attr.Value, 0); truncatedBytes != 0 {
```

Comment:
We believe that this limitation is no longer true in Shepherd (although it does need some testing to make certain that's the case). We should probably remove it -- could maybe be a different PR.

Reply:
Okay - I'll create a new ticket to investigate and we can remove in not required anymore.

Reply:
Okay - I've created #241 to investigate if we still need to truncate.